src/huggingface_hub/constants.py

```diff
@@ -25,7 +25,7 @@ def _as_int(value: Optional[str]) -> Optional[int]:
 
 # Constants for file downloads
 
-PYTORCH_WEIGHTS_NAME = "pytorch_model.bin"
+PYTORCH_WEIGHTS_NAME = "pytorch_model.safetensors"
```

Comment:
```suggestion
PYTORCH_WEIGHTS_NAME = "model.safetensors"
```

The convention on the HF Hub is to name this `model.safetensors`